utils/storage/s3.go

```diff
@@ -28,28 +30,39 @@ type S3FileStorage struct {
 	mutexes  *utils.KeyMutex
 }
 
+type customResolver struct {
```

Comment:
BaseEndpointだけで動く気がしていて、CustomResolverはいらないと思う
もともとカスタムリゾルバっぽくしてたのは、BaseURL書き換えるにはそうするしかなかったから


一応非推奨だし
> Because of this, we recommend that you don’t replace the EndpointResolverV2 implementation in your S3 client.